logging.conf

```diff
@@ -25,6 +25,12 @@ level=ERROR
 handlers=fileHandler
 qualname=robottelo.config
 
+[logger_robottelo_collection]
```

Comment:
@jyejare Check this out, added custom logger for collection issues.

Reply:
Looks Good ! ACKed!